modules/ROOT/pages/create-custom-namespace.adoc

```diff
@@ -12,13 +12,14 @@ When you deploy an application into an Anypoint Platform environment, Runtime Fa
 [source,copy]
 ----
 kubectl create ns <YOUR_ENVIRONMENT_NAME>
-kubectl label ns <YOUR_ENVIRONMENT_NAME> rtf.mulesoft.com/role=workers rtf.mulesoft.com/envId=<YOUR_ANYPOINT_PLATFORM_ENVIRONMENT_ID>
+kubectl label ns <YOUR_ENVIRONMENT_NAME> rtf.mulesoft.com/agentNamespace=rtf rtf.mulesoft.com/role=workers rtf.mulesoft.com/envId=<YOUR_ANYPOINT_PLATFORM_ENVIRONMENT_ID> rtf.mulesoft.com/org=<YOUR_ANYPOINT_PLATFORM_ORGANIZATION_ID>
 ----
 +
 * Replace `<YOUR_ENVIRONMENT_NAME>` with the name of your namespace.
```

Comment:
Can we also change all the upper cases for placeholders with lower cases so that this will be consistent throughout. 
`<YOUR_ENVIRONMENT_NAME>` to `<app_namespace>`
`<YOUR_ANYPOINT_PLATFORM_ENVIRONMENT_ID>` to `<environment_id>`
`<YOUR_ANYPOINT_PLATFORM_ORGANIZATION_ID>` to `<org_id>`

Reply:
It was all uppercase in the original document, but yeah maybe moving them all lowercase will be a better look.

Reply:
After trying it all with lowercase, if find that those parameters really pop with the caps. I think we should keep it.